TelepresenceBot/core/src/main/java/com/novoda/tpbot/bot/BotView.java

```diff
@@ -0,0 +1,13 @@
+package com.novoda.tpbot.bot;
+
+import com.novoda.tpbot.Direction;
+
+interface BotView {
```

Comment:
The plan is to use this in the follow-up presenters PR for the bot part of the app.